lib/routes/password.js

```diff
@@ -56,47 +58,47 @@ module.exports = function (
                 throw error.lockedAccount()
               }
               return checkPassword(emailRecord, oldAuthPW, request.app.clientAddress)
-              .then(
-                function (match) {
-                  if (!match) {
-                    throw error.incorrectPassword(emailRecord.email, form.email)
-                  }
-                  var password = new Password(
-                    oldAuthPW,
-                    emailRecord.authSalt,
-                    emailRecord.verifierVersion
-                  )
-                  return password.unwrap(emailRecord.wrapWrapKb)
-                }
-              )
-              .then(
-                function (wrapKb) {
-                  return db.createKeyFetchToken(
-                    {
-                      uid: emailRecord.uid,
-                      kA: emailRecord.kA,
-                      wrapKb: wrapKb,
-                      emailVerified: emailRecord.emailVerified
+                .then(
+                  function (match) {
+                    if (!match) {
+                      throw error.incorrectPassword(emailRecord.email, form.email)
                     }
-                  )
-                  .then(
-                    function (keyFetchToken) {
-                      return db.createPasswordChangeToken({
-                          uid: emailRecord.uid
-                        }
+                    var password = new Password(
+                      oldAuthPW,
+                      emailRecord.authSalt,
+                      emailRecord.verifierVersion
+                    )
+                    return password.unwrap(emailRecord.wrapWrapKb)
+                  }
+                )
+                .then(
+                  function (wrapKb) {
+                    return db.createKeyFetchToken(
+                      {
+                        uid: emailRecord.uid,
+                        kA: emailRecord.kA,
+                        wrapKb: wrapKb,
+                        emailVerified: emailRecord.emailVerified
+                      }
                       )
                       .then(
-                        function (passwordChangeToken) {
-                          return {
-                            keyFetchToken: keyFetchToken,
-                            passwordChangeToken: passwordChangeToken
-                          }
+                        function (keyFetchToken) {
+                          return db.createPasswordChangeToken({
+                              uid: emailRecord.uid
+                            }
+                            )
```

Comment:
it's not clear at a glance whether the logic of this function has changed, or just the indentation